examples/panantarctic_regional_simulation.jl

```diff
@@ -0,0 +1,237 @@
+using ClimaOcean
+using ClimaOcean.ECCO
+using Oceananigans
+using Oceananigans.Units
+using Oceananigans.Grids: φnode
+using CairoMakie
+using CFTime
+using Dates
+using Printf
+using CUDA
+
+Oceananigans.TimeSteppers.time_step!(::Nothing, dt) = nothing
```

Comment:
do we need this?

Reply:
I think we need the `using CUDA` since that's how GPUs work now adays, or so I thought.

I don't think the other two are needed through.  I will try it tomorrow without the three and see if it still runs.

Reply:
I only meant to comment on the

```Julia
Oceananigans.TimeSteppers.time_step!(::Nothing, dt) = nothing
```